sys/kern/src/arch/riscv32/mtimer.rs

```diff
@@ -0,0 +1,31 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is odd, the license is already there...

Reply:
The only difference that I see between the license line in this file and other files (as well as the suggestion) is a period at the end of the line i.e. MPL/2.0/. instead of MPL/2.0/

Reply:
So that was it?

Reply:
yea.